styles.css

```diff
@@ -369,3 +371,188 @@ form {
 .btn-body-footer:disabled {
   border: 1px solid #c1c7d0;
 }
+
+.menu-bar img {
+  display: block;
+}
+
+.menu-bar ul {
+  display: none;
+}
+
+.work-bottom2 {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .menu-bar ul {
+    text-transform: capitalize;
+    display: block;
+    display: flex;
+    align-items: center;
+    padding: 0 3rem;
+  }
+
+  .menu-bar li {
+    padding: 0 0 0 1.5rem;
+  }
+
+  .desktop-menu a {
+    color: #6070ff;
+  }
+
+  .img-logo {
+    padding: 0 3rem;
+  }
+
+  .menu-bar img {
+    display: none;
+  }
+
+  .intro h1 {
+    margin: 0 5rem;
+    width: 80%;
+    word-spacing: 0.05rem;
+  }
+
+  .intro p,
+  .connect-header,
+  .icons-header {
+    margin: 0 5rem;
+    word-spacing: 0.05rem;
+    line-height: 2rem;
+  }
+
+  .about-myself-top {
+    background: #fff url(img/desktop-header-bg.svg) 50% no-repeat;
+    background-size: contain;
+  }
+
+  .work {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr;
+    margin: 0 auto;
+    align-items: center;
+  }
+
+  .img-work {
+    padding: 1rem 1rem 1rem 1rem;
+
+    /* padding:0; */
+    width: 100%;
+  }
+
+  .body-content {
+    margin: 1rem;
+  }
+
+  .work-top {
+    margin: 0 1rem 1.5rem 1rem;
+  }
+
+  .work-top li {
+    padding: 0 0.4rem 0 0;
+  }
+
+  .work-bottom1 {
+    display: none;
+  }
+
+  .work-bottom2 {
+    display: flex;
+    align-self: flex-start;
+    padding: 1rem 2rem 0 1rem;
+  }
+
+  .work-bottom2 li {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1rem;
+    color: #6070ff;
+    border: 1px solid #dadbff;
+    border-radius: 0.5rem;
+    padding: 0 0.2rem;
+    margin-right: 1rem;
+    background-color: #ebebff;
+  }
+
+  .about-myself,
```

Comment:
Please use the same font color for heading, languages, frameworks, and skills as in the design.